### PR TITLE
Fixes in documentation commands

### PR DESCRIPTION
**Pull request by susuhahnml**

### CODE REVIEW

File: doc/content/installation.md [hunk 1]
```diff
@@ -4,17 +4,17 @@ fillname requires Python 3.8+. We recommend version 3.10.
 
 You can check a successful installation by running
 
-```shell
-fillname -h
+```console
+$ fillname -h
```
Comment (sthiele) [inline doc/content/installation.md:8]: Hi @susuhahnml, why did you choose to use console and dollar sign here? I actually like the version in the README better.
The dollar sign makes it harder to just copy the code into the terminal.

File: README.md [hunk 2]
```diff
@@ -11,13 +11,13 @@
...
+```bash
 pip install .
 ```
 
 ## Usage
 
-```shell
+```bash
 fillname -h
 ```
```
Comment (sthiele) [inline README.md:22]: @susuhahnml, this version. :wink:

File: doc/content/installation.md (hunk 1 above)
Comment (rkaminsk) [reply doc/content/installation.md:8]: I sometimes use the `$` sign to distinguish code and output. If there is just code, then there is no need for the `$`.

Comment (susuhahnml) [reply doc/content/installation.md:8]: In the documentation the copy button will ignore this sign. And make a nicer format.

Comment (susuhahnml) [reply doc/content/installation.md:8]: If you don't want to compile the documentation, just check out the clinguin or clingraph read the docs as an example

Comment (sthiele) [reply doc/content/installation.md:8]: I just checked the clinguin example, and it did not work for me. The dollar sign was also copied. :cry:

https://clinguin.readthedocs.io/en/latest/clinguin/use.html

```console
$ clinguin server --domain-files examples/angular/sudoku/instance.lp examples/angular/sudoku/encoding.lp --ui-files examples/angular/sudoku/ui.lp
```

Comment (susuhahnml) [reply doc/content/installation.md:8]: Hey! Sorry, the clinguin docu aparently is not updated. Check the clingraph one: https://clingraph.readthedocs.io/en/latest/clingraph/installation.html 

With this package the prompt should not be copied.

Comment (rkaminsk) [reply doc/content/installation.md:8]: Is that why you added the complicated regexp? I admit I did not try to understand what is happening there. It somehow seems a lot of effort just to get a dollar sign.

Comment (susuhahnml) [reply doc/content/installation.md:8]: Yes. Because it covers other prompts. We can simplify it and sick with one, `>` or `$`

Comment (rkaminsk) [reply doc/content/installation.md:8]: This comes back to Sven's comment. Why have it at all? One could simply write the commands without a prefix and does not even need a complicated copy button. I'll leave it for you to decide. Just my two cents.

Comment (susuhahnml) [reply doc/content/installation.md:8]: Personally I like to have the prompt. I think is more intuitive.

Comment (sthiele) [reply doc/content/installation.md:8]: Yes, that worked :+1:

**Pull request merged**